### DISCUSSION TITLE
Does Fabric v6 fire eventjs events

### DISCUSSION THREAD

**Discussion by hamzawain7** [fabricjs/fabric.js#9959] [Q&A]
I see [here](https://github.com/fabricjs/fabric.js/blob/master/src/canvas/Canvas.ts#L186) the commented-out code in canvas.ts, and ts is also throwing an error when I try to listen to events like 'touch:longpress'. Was eventjs functions left out? what can I do to get them back?

    // if (typeof eventjs !== 'undefined' && eventjsFunctor in eventjs) {
    //   eventjs[eventjsFunctor](canvasElement, 'gesture', this._onGesture);
    //   eventjs[eventjsFunctor](canvasElement, 'drag', this._onDrag);
    //   eventjs[eventjsFunctor](
    //     canvasElement,
    //     'orientation',
    //     this._onOrientationChange
    //   );
    //   eventjs[eventjsFunctor](canvasElement, 'shake', this._onShake);
    //   eventjs[eventjsFunctor](canvasElement, 'longpress', this._onLongPress);
    // }

**Comment by asturur** [comment]
No eventjs touch controls are for now disabled because eventjs is unsupported and not updated since long, and we don't really have an alternative so far.
